tests/foreman/cli/test_discoveredhost.py

```diff
@@ -103,7 +103,7 @@ def test_rhel_pxeless_discovery_provisioning(
     :Setup: Provisioning should be configured and a host should be
         discovered via cli
 
-    :expectedresults: Host should be provisioned successfully
+    :expectedresults: Host should be provisioned successfully.
```

Comment:
Why? This isn't even for the sake of consistency. The other tests don't have period here as well.